tests/functional/test_eclab_parser.py

```diff
@@ -318,6 +317,15 @@ def test_shape(measurements_with_data):
     of the columns are the same.
 
     """
+    if measurements_with_data[0].name in (
+        "multiple_techniques_dataset_01_02_CVA_C01.mpt",
+        "multiple_techniques_dataset_01_07_ZIR_C01.mpt",
+        "multiple_techniques_dataset_01_08_CVA_C01.mpt",
+        "dataset_with_loop_01_01_OCV_DUSB0_C01.mpt",
+        "dataset_with_loop_01_02_CVA_DUSB0_C01.mpt",
+    ):
+        skip("TEMP SKIP. BROKEN, SEE: https://github.com/ixdat/ixdat/issues/158")
```

Comment:
Wait a second @KennethNielsen , these have to to with .mpt reading, so not related to #158. What goes wrong here?